library/general/test/test_helper.rb

```diff
@@ -3,13 +3,8 @@
 ENV["Y2DIR"] = inc_dirs.join(":")
 
 require "yast"
+require_relative "scr_stub"
 
-def assign_root_path(directory)
-  check_version = false
-  handle = Yast::WFM.SCROpen("chroot=#{directory}:scr", check_version)
-  Yast::WFM.SCRSetDefault(handle)
-end
-
-def reset_root_path
-  Yast::WFM.SCRClose(Yast::WFM.SCRGetDefault)
+RSpec.configure do |c|
```

Comment:
Is this block needed? It is already present in `scr_stub.rb` file which is required here.